CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
i've never used this.  are you recommending that it should be on in master?  or did it slip in by accident?